src/components/InputControl/InputControl.js

```diff
@@ -0,0 +1,37 @@
+import React, { forwardRef, useState } from "react";
+import PropTypes from "prop-types";
+
+import { Eye, EyeOff } from "react-feather";
+
+import styles from "./InputControl.module.scss";
+
+const InputControl = forwardRef((props, ref) => {
+  const [visible, setVisible] = useState(props.password ? false : true);
+  return (
+    <div className={styles.container}>
+      {props.label && <label className={styles.label}>{props.label}</label>}
+      <div className={styles.inputContainer}>
+        <input
+          className="basic-input"
+          type={visible ? "text" : "password"}
+          {...props}
+          ref={ref}
```

Comment:
did you checked if forward ref is working with this component or not ?

Reply:
yes